In [1]:
import pandas as pd
import numpy as np
import math

1) Известно, что генеральная совокупность распределена нормально
со средним квадратическим отклонением, равным 16.
Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95,
если выборочная средняя M = 80, а объем выборки n = 256.

Доверительным называется интервал, который с заданной надежностью $\alpha $ покрывает оцениваемый параметр.

In [2]:
T1=80-1.96*16/math.sqrt(256)
T1

78.04

In [3]:
80+1.96*16/math.sqrt(256)

81.96

T1 < $\alpha $ < T2

78.04 <  𝛼  < 81.96

2) В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,
получены опытные данные:
6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1
Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,
оценить истинное значение величины X при помощи доверительного интервала, покрывающего это
значение с доверительной вероятностью 0,95.

In [4]:
array = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]

In [5]:
x=sum(array)/len(array)
x

6.589999999999999

In [6]:
S=np.std(array,ddof=1)
S

0.4508017549014448

In [7]:
t=2.262

In [10]:
T1=x-t*S/math.sqrt(10)
T1

6.267538255912425

In [11]:
T1=x+t*S/math.sqrt(10)
T1

6.912461744087573

Истинное значение случайной величины X лежит в интервале 6.26 < 𝛼 < 6.91  с доверительной вероятностью 0,95

3.) Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.
Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр
оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

In [12]:
Zn=(17.5-17)/(4/math.sqrt(100))
Zn

1.25

In [13]:
Zt=1.645

Zn < Zt (1,25 < 1.645) Значит H0 верна на уровне значимости 0,05.

4) Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [14]:
array = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]

In [15]:
x=sum(array)/len(array)
x

198.5

In [16]:
S=np.std(array,ddof=1)
S

4.453463071962462

In [17]:
Tn=(x-200)/(S/math.sqrt(10))
Tn

-1.0651074037450896

In [18]:
Tt=3.25

|Tn|=1.065 < 3.25=Tt,  то нулевую гипотезу о равенстве среднего веса можно принять. при уровне значимости 0,01